src/ApiCompat/Microsoft.DotNet.ApiCompatibility/IAssemblySymbolLoader.cs

```diff
@@ -1,7 +1,9 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#if NET472_OR_GREATER
```

Comment:
@vlada-shubina curious, why was this change needed?

Reply:
good catch. This directive can be removed. 
Strangely enough it is not shown as an error in full framework, only in .NET.

Reply:
I'll fix it in another PR shortly.

Reply:
Fixed in https://github.com/dotnet/sdk/pull/27795/commits/1b030c7591e7f866530fcc1f11cc8dce71d98387